tests/generic_config_updater/test_mgmt_interface.py

```diff
@@ -16,6 +16,25 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture(autouse=True)
+def ignore_expected_loganalyzer_exceptions(duthosts, rand_one_dut_hostname, loganalyzer):
```

Comment:
is it possible to reuse https://github.com/sonic-net/sonic-mgmt/blob/master/ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt